src/main/java/org/dependencytrack/tasks/vulnerabilitypolicy/blobstorage/BlobStorageAccessHandler.java

```diff
@@ -0,0 +1,26 @@
+package org.dependencytrack.tasks.vulnerabilitypolicy.blobstorage;
+
+
+import io.minio.errors.ErrorResponseException;
+import io.minio.errors.InsufficientDataException;
+import io.minio.errors.InternalException;
+import io.minio.errors.InvalidResponseException;
+import io.minio.errors.ServerException;
+import io.minio.errors.XmlParserException;
+
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.zip.ZipInputStream;
+
+public interface BlobStorageAccessHandler {
+
+    default boolean verifyDownloadNeeded() throws ServerException, InsufficientDataException, ErrorResponseException, IOException, NoSuchAlgorithmException, InvalidKeyException, InvalidResponseException, XmlParserException, InternalException {
+        return true;
+    }
+
+    default ZipInputStream downloadZippedContent() throws IOException, ServerException, InsufficientDataException, ErrorResponseException, NoSuchAlgorithmException, InvalidKeyException, InvalidResponseException, XmlParserException, InternalException {
+        return null;
+    }
```

Comment:
These interface methods should not throw MinIO exceptions. Try to reduce it to 1-2 generic exceptions (or create a custom exception class), for example `IOException`. Implementations will then need to re-throw their internal exceptions accordingly.